iree/compiler/Dialect/LinalgExt/IR/LinalgExtOps.td

```diff
@@ -253,38 +253,45 @@ def LinalgExt_ReverseOp : LinalgExt_Op<"reverse", [
                             ["payloadUsesValueFromOperand"]>]> {
   let summary = "Reverse operator";
   let description = [{
-    A temporary solution of a reverse op. The loop bound of the reverse
-    dimension is half of the shape because we can simply swap elements. E.g.,
-
-    for (int i = 0; i < n / 2; ++i) {
-      std::swap(a[i], a[n - 1 - i]);
+    A temporary solution of a reverse op, which allows IREE tiles and
```

Comment:
Nit : replace with 
`A temporary solution for lowering reverse ops into IREE, allowing IREE to tile and distribute them`

Reply:
Thanks for the better wording!